libraries/standard/json/cbmc/sources/skipGeneric.c

```diff
@@ -0,0 +1,33 @@
+#include "skipGeneric.h"
+
+/**
+ * See skipGeneric.h for docs
+ *
+ * Advance buffer index beyond some minimum value.
+ */
+static bool_ skipGeneric( const char * buf,
+                          size_t * start,
+                          size_t max,
+                          size_t min )
+{
+    bool_ ret = nondet_bool() ? true : false;
+
+    if( ret == true )
+    {
+        size_t x;
+        __CPROVER_assume( x >= min );
+        __CPROVER_assume( x <= max );
+        __CPROVER_assume( *start <= max );
```

Comment:
Should we __CPROVER_assert that start is not NULL?

Reply:
sure, I'll add that.